composer.json

```diff
@@ -38,6 +38,7 @@
         "kreait/firebase-tokens": "^3.0",
         "lcobucci/jwt": "^4.1",
         "mtdowling/jmespath.php": "^2.6.1",
+        "phpfastcache/phpfastcache": "^9.1",
```

Comment:
See https://github.com/kreait/firebase-php/pull/747#issuecomment-1304133629 for why I would like to have phpfastcache removed again 😅 

Could you instead please add `firebase/php-jwt` as a direct dependency? At the moment it's transitively included through the Google Auth library, but if they were to remove it, we wouldn't have it anymore either.